modules/Kconfig.mcux

```diff
@@ -366,18 +366,21 @@ config HAS_MCUX_XBARA
 
 config NXP_FW_LOADER
 	bool "Include firmware loader component"
+	default y if (BT || WIFI)
```

Comment:
The firmware loader is not used on all NXP BLE platforms, currently only RW uses it.
MCXW71 loads its firmware through the ROM, using nxp blhost tool.
So, I think we should keep it disabled by default and enable it in the soc defconfig.
Or maybe `default y if SOCXXX` but this could become kinda bloated ?